src/index.ts

```diff
@@ -0,0 +1,4 @@
+import * as synthetics from './commands/synthetics'
+import {parseConfigFile} from './helpers/utils'
+
+export {parseConfigFile, synthetics}
```

Comment:
I'd namespace `utils`.
```suggestion
import * as utils from './helpers/utils'

export {utils, synthetics}
```

WDYT?

Reply:
I thought that the config file is important enough to have some function in the root namespace or a `config` namespace, rather than pushed in a `util` namespace which might get crowded and doesn't convey a strong meaning.
No strong opinion, though.